packages/editor-ui/src/App.vue

```diff
@@ -79,7 +80,7 @@ export default mixins(
 			}
 		},
 		logHiringBanner() {
-			if (!this.isHiringBannerEnabled && this.$route.name !== VIEWS.DEMO) {
+			if (this.isHiringBannerEnabled && this.$route.name !== VIEWS.DEMO) {
```

Comment:
why did you change logic here?

Reply:
With the original implementation, `export N8N_HIRING_BANNER_ENABLED=false` does not disable the banner.